internal/avs/config.go

```diff
@@ -18,16 +18,19 @@ type Config struct {
 	Disabled                                        bool   `envconfig:"default=false"`
 	InternalTesterAccessId                          int64
 	InternalTesterService                           string `envconfig:"optional"`
-	InternalTesterTags                              []*Tag `envconfig:"optional"`
 	GroupId                                         int64
 	ExternalTesterAccessId                          int64
 	ExternalTesterService                           string `envconfig:"optional"`
-	ExternalTesterTags                              []*Tag `envconfig:"optional"`
+	ExternalTesterDisabled                          bool   `envconfig:"default=false"`
```

Comment:
This is not being used, do you plan to introduce the possibility of  disabling the external avs evaluation in another PR?

Reply:
the original task was not included this to be removed, only the internal/external tags and gardenerseedNameTags. if the second task (which is about to remove avs external evaulations, then in another PR i will remove this).